js/src/views/ParityBar/accountStore.js

```diff
@@ -33,7 +35,15 @@ export default class AccountStore {
   }
 
   @action setDefaultAccount = (defaultAccount) => {
-    this.defaultAccount = defaultAccount;
+    transaction(() => {
+      this.accounts = this.accounts.map((account) => {
+        account.default = account.address === this.defaultAccount;
```

Comment:
Is this correct? It basically sets the default to the previous default, i.e. this.defaultAccount is previous version since default is only set afterwards. Looks wrong. (If correct, a NOTE explanation may be in order here)